common/pom.xml

```diff
@@ -36,6 +36,11 @@ under the License.
       <artifactId>reef-io</artifactId>
       <version>${reef.version}</version>
     </dependency>
+    <dependency>
```

Comment:
why do we need this here?

Reply:
Without this line, we use the commons-lang3 library from `reef-common`, which uses commons-lang3 library from 2014. Since our project uses commons-lang3 library quite often, my opinion is that it's worth upgrading.